src/schema/v2/me/sendIdentityVerficationEmailMutation.ts

```diff
@@ -119,7 +125,16 @@ export const sendIdentityVerificationEmailMutation = mutationWithClientMutationI
     }
 
     return sendIdentityVerificationEmailLoader({ user_id: userID, email, name })
-      .then((result) => result)
+      .then((result) => {
+        const forceUrl = process.env["PRODUCTION_ENV"]
+          ? "https://artsy.net"
```

Comment:
These values should likely come from an ENV var itself.

Reply:
will add